nx/lib/nx.ex

```diff
@@ -5319,6 +5362,7 @@ defmodule Nx do
     def unquote(name)(tensor) do
       case to_tensor(tensor) do
         %T{type: {type, _}} = tensor when type in [:s, :u] -> tensor
+        %T{type: {:c, _}} -> Nx.Shared.raise_complex_not_supported(unquote(name), 1)
```

Comment:
Is it not supported now or not supported permanently? If not supported now, I would keep it working here and let it raise elsewhere.

Reply:
Ignore me. This was answered later.